main.cpp

```diff
@@ -50,87 +50,95 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
-
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: monster
+//  action 1: the monster attacks
+monster.attack();
+//  action 2: the monster eats
+monster.eat();
+//  action 3: the monster dances
+monster.attack();
+    
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: television
+//  action 1: the television plays a movie
+television.playMovie();
+//  action 2: the television consumes power
+television.consumePower();
+//  action 3: the television syncs with a bluetooth device
+television.syncWithBluetoothDevice();
+    
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: saw
+//  action 1: the saw cuts wood
+saw.cutWood();
+//  action 2: the saw makes noise
+saw.makeNoise();
+//  action 3: the saw inspires fear
+saw.inspireFear();
+    
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: human
+//  action 1: the human puts on gloves
+human.putOnGloves();
+//  action 2: the human cook a ham
+human.cookHam();
+//  action 3: the human buys a papaya
+human.buyPapaya();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: engine
+//  action 1: the engine rotates
+engine.rotate();
+//  action 2: the engine burns fuel
+engine.burnFuel();
+//  action 3: the engine makes noise
+engine.makeNoise();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: baby
+//  action 1: the baby cries
+baby.cry();
+//  action 2: the baby crawls
+baby.crawl();
+//  action 3: the baby spits up
+baby.spitUp();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: boss
+//  action 1: The boss hires
+boss.hire();
+//  action 2: the boss fires
+boss.fire();
+//  action 3: the boss sleeps
+boss.sleep();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.